src/components/share/Default.jsx

```diff
@@ -0,0 +1,103 @@
+import React, { useRef, useState } from 'react';
+import { Image, View } from 'react-native';
+import CommonBottomSheet from '../base/modal/CommonBottomSheet';
+import Button from '../base/Button';
+import Card from '../base/Card';
+import Folder from '../../assets/icons/Folder';
+import Search from '../../assets/icons/Search';
+import size from '../../utils/size';
+import styled from 'styled-components';
+import { ScrollView, TouchableOpacity } from 'react-native-gesture-handler';
+
+const Default = () => {
+  const bottomSheetRef = useRef(null);
+
+  const openBottomSheet = () => {
+    bottomSheetRef.current?.expand();
+  };
+
+  const ref = useRef(null);
+
+  return (
+    <CommonBottomSheet
+      ref={bottomSheetRef}
+      snapPoints={['58%']}
+      title="폴더 선택"
+      leftButtonType="none"
+      rightButtonType="close"
+      onRightButtonPress={() => console.log('Close button pressed')}
```

Comment:
console.log 사용을 지양해주세요. 이하동문

```suggestion
      onRightButtonPress={() => null}
```

Reply:
으악 넵